pkg/watchdog/interface.go

```diff
@@ -11,6 +11,8 @@ type Watchdog interface {
 	Start(ctx context.Context) error
 	// IsStarted return if the watchdog is running
 	IsStarted() bool
+	// IsRebooting return if the watchdog has already stopped feeding and currently about to reboot the node
```

Comment:
`IsStopped` would be a better name IMHO, because watchdogs can do more than just reboots

Reply:
My problem with `IsStopped` is that it can be interpreted in two different ways:
1. A previously running watchdog was stopped, so now it `isStopped`
2. An existing watchdog was never started therefore it `isStopped`

I'm looking for a meaningful names that will only match the first option.

Reply:
`isTriggered` or `isTerminating` perhaps?

Reply:
>isTriggered or isTerminating perhaps?

Thanks !
both SGTM, @slintes do you have any preference ?

Reply:
Yep, I like this idea.
Done.